backend/Models/CinemaRoom.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace CinemaApp.Models
+{
+    public class CinemaRoom
```

Comment:
as this is a Data Model in the database (a different table) please add an Id to it.